Bottom: 50dfd05699adca9abcbb4b4be2ae9efe2a665bf6
Top:    3a637cce50448fe5e3993a348adcd5541c82344f
Author: aingerson <alexia.ingerson@intel.com>
Date:   2018-11-27 13:48:54 -0800

Refresh of fabtests_repo-0

---

diff --git a/single-node.sh b/single-node.sh
index 8232b22..3dd6901 100644
--- a/single-node.sh
+++ b/single-node.sh
@@ -21,7 +21,10 @@ sudo make install
 
 # Uses the fabtests branch corresponding to the PR's target branch
 echo "==> Building fabtests"
-cd $WORKSPACE/libfabric/fabtests
+cd $WORKSPACE
+git clone https://github.com/ofiwg/fabtests
+cd fabtests
+git checkout $TARGET_BRANCH
 ./autogen.sh
 ./configure --with-libfabric=$WORKSPACE/libfabric/install/ \
 		--prefix=$WORKSPACE/fabtests/install/ \
@@ -30,15 +33,7 @@ make -j 4
 sudo make install
 
 # Runs all the tests in the fabtests suite while only expanding failed cases
-declare EXCLUDE=$WORKSPACE/fabtests/install/share/fabtests/test_configs/$PROVIDER/${PROVIDER}.exclude
-if [ -f $EXCLUDE ]; then
-	EXCLUDE="-R -f $EXCLUDE"
-else
-	EXCLUDE=""
-fi
-
 echo "==> Running fabtests"
 LD_LIBRARY_PATH=$WORKSPACE/fabtests/install/lib/:$LD_LIBRARY_PATH	\
 BIN_PATH=$WORKSPACE/fabtests/install/bin/ FI_LOG_LEVEL=debug		\
-$WORKSPACE/fabtests/install/bin/runfabtests.sh -v $EXCLUDE		\
-$PROVIDER 127.0.0.1 127.0.0.1
+$WORKSPACE/fabtests/install/bin/runfabtests.sh -v $PROVIDER 127.0.0.1 127.0.0.1
